src/components/BookActionButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useBookDetailsContext } from "../../store/books/details/BookDetailsContext";
+import ConfirmationModal from "../Modals/ConfirmationModal";
+import BookRentButton from "./BookRentButton";
+import BookReturnButton from "./BookReturnButton";
+
+const confirmRentingModalTitle = "Please confirm book rental";
+const confirmRentingModalDescription =
+  "After confirmation, you will receive an email with details of the book rental";
+
+const confirmReturnModalTitle = "Please confirm book return";
+const confirmReturnModalDescription =
+  "After confirmation, you will receive an email with confirmation of book return";
+
+type Props = {
+  status?: string;
+};
+
+const BookActionButton = ({ status }: Props) => {
```

Comment:
We should continue with practice of grouping the code by the rent/return action.
```
<BookRentButton />
<ConfirmationModal
....
/> 
```
could be extracted to another component "BookRentAction", and another one to "BookReturnAction". 
That way all the text won't be handled in the same place (confirmRentingModalTitle, confirmRentingModalDescription, confirmReturnModalTitle, confirmReturnModalDescription).

Remember that this kind of things have potential to grow.

Reorganize Components folder structure so that there is:

components
   BookAction
        BookRentAction
              BookRentButton
        BookReturnAction
              BookReturnButton
        index.ts

Reply:
When comparing status to "AVAILABLE" or "RENTED_BY_YOU" or so, use already existing Enum